x/distribution/keeper/keeper.go

```diff
@@ -35,123 +35,53 @@ func NewKeeper(cdc *codec.Codec, key sdk.StoreKey, paramSpace params.Subspace, c
 	return keeper
 }
 
-//______________________________________________________________________
-
-// get the global fee pool distribution info
-func (k Keeper) GetFeePool(ctx sdk.Context) (feePool types.FeePool) {
-	store := ctx.KVStore(k.storeKey)
-	b := store.Get(FeePoolKey)
-	if b == nil {
-		panic("Stored fee pool should not have been nil")
+// withdraw rewards from a delegation
+func (k Keeper) WithdrawDelegationRewards(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) sdk.Error {
```

Comment:
The placement of this function is counterintuitive to me - should it not go near the unexposed `WithdrawDelegationRewards` function? I'd imagine this'd be good in a `delegation.go` file as previously suggested.

Reply:
Hmm really? I like the separation of all the internal calculation functions into `delegation.go` / `validator.go`, the exposed functions (called directly by handlers) in `keeper.go`, and all the hooks (which just call internal functions) in `hooks.go`.

Reply:
okay - it's alright, I see the logic